clang/runtime/dpct-rt/include/dpct/math.hpp

```diff
@@ -2055,6 +2055,102 @@ class joint_matrix {
   matrix_accessor x;
   const size_t num_elements;
 };
+
+/// Stores 1 8x8 b16 matrix from local memory to shared memory (32-bits per wi)
```

Comment:
Please use sycl language in the comments

Reply:
Done